doc/howto/network_load_balancers.md

```diff
@@ -63,7 +62,8 @@ Use the following command to add a backend specification:
 lxc network load-balancer backend add <network_name> <listen_address> <backend_name> <listen_ports> <target_address> [<target_ports>]
 ```
 
-The target ports are optional, if not specified, then the load balancer will use the listen ports for the backend.
+The target ports are optional.
+If not specified, the load balancer will use the listen ports for the backend.
```

Comment:
By breaking this into two points you need to restate that the 2nd point is talking about target ports.

Reply:
" for the backend target ports"

Reply:
Not sure I agree that this is unclear otherwise, but added. :)

Reply:
Because its on a new line it reads to me like you have moved on to a different subject (and the section topic isn't about target ports specifically). If it were on the same line it would have read fine together.

Reply:
In the output it is on the same line. But it definitely doesn't hurt to be more specific. :)

Reply:
Why did you move it onto separate lines in the source btw?

Reply:
I'm used to starting every sentence on a new line, because it makes it easier to review - when changing the language, it'll usually affect a sentence, not a "random" 72-character line.
That's no hard rule, but you did the same in this doc. ;)
In the output, consecutive lines are displayed as one paragraph, so it doesn't make a difference.

And grammar-wise, it had to be two sentence, since you can't join two complete sentences with a comma.

Reply:
If I did it was coincidence rather than by design, probably to avoid long line lengths or because I copied the OVN forward docs as a basis for this one. :)
The original probably could have done with an "and" in it to make it flow more clearly.